CoinDock-app/src/Screens/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from "react";
+
+import "Shared/common-styles/space.css";
+import "./Dashboard.css";
+import { PieChart } from "Shared/Chart/Doughnut/Doughnut";
+import { LineChart } from "Shared/Chart/LineChart/LineChart";
+import "Shared/common-styles/space.css";
+import Cards from "Shared/Section2/Cards";
+import Wallet from "Screens/Wallets/Wallet";
+import { useTopperformer } from "App/Api/CoinPerformence/coinperformance";
+import { useLowperformer } from "App/Api/CoinPerformence/coinperformance";
+import { usePrimaryCurrency } from "App/Api/CoinPerformence/coinperformance";
+import { useTotalCurrency } from "App/Api/CoinPerformence/coinperformance";
+import DownArrow from "Shared/images/downarrow.jpg";
+import UpArrow from "Shared/images/uparrow.png";
+
+function Dashboard() {
+  const [isLoaded, setIsLoaded] = useState(false);
+  const { data: total } = useTotalCurrency();
+  const { data: primary } = usePrimaryCurrency();
+  const { data: top } = useTopperformer();
+  const { data: low } = useLowperformer();
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Why loading is set with the timeout can we use loading value from the hooks and we can set it.

Reply:
Okay